openedx/core/djangoapps/django_comment_common/models.py

```diff
@@ -267,6 +268,27 @@ def divided_discussions(self, value):
         """
         self._divided_discussions = json.dumps(value)
 
+    @request_cached()
```

Comment:
I suspect we needed this in the first place because the system was previously setup to perform 2 `get`s for each update. Anyway, I'm leaving it in place, just in case.